OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs

```diff
@@ -52,33 +58,29 @@ public PowerDownBotASModule(Actor self, PowerDownBotASModuleInfo info)
 			world = self.World;
 			player = self.Owner;
 			toggledBuildings = new List<BuildingPowerWrapper>();
-			isToggledBuildingsValid = a => a.Owner == self.Owner && !a.IsDead && a.IsInWorld && GetTogglePowerChanging(a) < 0;
+			isToggledBuildingsValid = a => a.Owner == self.Owner && !a.IsDead && a.IsInWorld;
 		}
 
 		protected override void Created(Actor self)
 		{
-			// Special case handling is required for the Player actor.
```

Comment:
Why are you changing this? I'm not that confident in changing this one.

Reply:
https://github.com/OpenRA/OpenRA/commit/aeab9a8116f51e7f467f37904658f7c22bc57276

So I changed that.

Reply:
Ah, https://github.com/OpenRA/OpenRA/pull/18182. Okay, that one I overlooked.